composer.json

```diff
@@ -39,7 +39,8 @@
 		}
 	],
 	"require": {
-		"php": ">=5.3.3",
+		"php": ">=5.4",
+		"phpbb/phpbb": ">=3.2.0@dev",
```

Comment:
Well as for this PR, I need to merge this so I can get my real feature PR made, so should we remove this for now or what?